### DISCUSSION TITLE
hide search bar in backpack

### DISCUSSION THREAD

**Discussion by phpdev-ed** [Laravel-Backpack/community-forum#505] [General]
how can i hide search bar in crud page for specific crud?

**Comment by jcastroa87** [comment]
Hello @phpdev-ed 

You can create a custom view and set this by

```
$this->crud->setListView('path.to.your.view');
```

You can check our [documentation](https://backpackforlaravel.com/docs/6.x/crud-api#custom-views).

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
